executor/executor.yml

```diff
@@ -38,8 +38,8 @@
   register: c_env_length
 
 - set_fact:
-    envNameList: ["APP_NAMESPACE", "APP_LABEL", "APP_KIND", "CHAOSENGINE"]
-    envValueList: ["{{ c_app_ns }}", "{{ c_app_label }}", "{{ c_app_kind }}", "{{ c_engine }}"]
+    envNameList: ["APP_NAMESPACE", "APP_LABEL", "APP_KIND", "CHAOSENGINE","AUXILIARY_APPINFO"]
+    envValueList: ["{{ c_app_ns }}", "{{ c_app_label }}", "{{ c_app_kind }}", "{{ c_engine }}, {{ c_auxiliary_appinfo }}"]
```

Comment:
@shubhamchaudhary  typo in envValueList missed quotes (") for c_engine and c_auxiliary_appinfo